### DISCUSSION TITLE
Help needed: "Failed to create the part's controls" -- [Resolved]

### DISCUSSION THREAD

**Discussion by Physce** [mytourbook/mytourbook#1162] [Q&A]
Good morning,

I have a problem with MyTourbook:
I had an old version that no longer wanted to work, so I installed the latest version (23.5.0).
This version starts but shows me the error message "Failed to create the part's controls" in the "parcours import" window.
So I uninstalled java and installed the latest version, it didn't change anything.
I am on Windows 10.

Has this ever happened to anyone?

Thank you in advance for your help.

Simon.

**Comment by FJBDev** [comment]
Please see [this thread](https://github.com/mytourbook/mytourbook/issues/1133) as I think it's a duplicate of the issue that was fixed in 23.5.1

**Comment by Physce** [comment]
It's possible yes.

If I understand correctly I have to install equinox to have "p2" to install version 23.5.1 !?
it's beyond my skills.... i don't even understand how to install eclipse -_-

**Reply by FJBDev** [reply]
No need to install p2. [This](https://mytourbook.sourceforge.io/mytourbook/index.php/download-install/software-updates) should help to guide you.

**Reply by Physce** [reply]
OK, I obviously didn't understand.

I tried to upgrade to version 23.5.1 as indicated on the MyTourBook site but it didn't work, error message during installation.

**Reply by Physce** [reply]
![MyTourBook_23 5 1](https://github.com/mytourbook/mytourbook/assets/141484158/a84aa216-a7c3-441c-9855-063f47524619)

**Reply by Physce** [reply]
Ok, I tried again and it worked!

It seems that solved the problem.

Thank you so much !!!

**Reply by Physce** [reply]
I confirm, it works again. I just have a small inconsequential error message that appears when I import a route.

**Reply by FJBDev** [reply]
Glad to know it works again!!

Just curious, what are the error messages you are getting?

**Reply by Physce** [reply]
![MyTourBook_23 5 1b](https://github.com/mytourbook/mytourbook/assets/141484158/10d09df9-63f9-45b6-a617-fef117ad74a8)

**Reply by FJBDev** [reply]
Ok, that seems to be because maybe your didn't enter any SRTM credentials ? Se the documentation [here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/preferences/pref-srtm/pref-srtm-data)

If it's a different issue, then please create a separate issue as it is different than your original issue

**Reply by Physce** [reply]
Correct, there was no SRTM. Everything is in order now.
Thank you.

**Reply by FJBDev** [reply]
Glad to know that was the issue !
